v2/catalog-shacl.shce

```diff
@@ -0,0 +1,147 @@
+PREFIX con: </catalog-skos.ttl#>
```

Comment:
I think we shouldn't include `.ttl` in the URI. This might need to wait for v3 since it is a breaking change.

https://www.w3.org/2001/tag/2003/webarch-20031027/#uri-opacity

> The `.html` at the end of `http://example.com/page.html` provides no guarantee that representations of the identified resource will be served with the Internet Media Type `text/html`. The HTTP protocol does not constrain the Internet Media Type based on the path component of the URI; the server is free to return a representation in the PNG or any other format for that URI.